docs/reference-python/contexts.md

```diff
@@ -0,0 +1,233 @@
+---
+sidebar_position: 2
+title: Contexts
+description: API reference for DBOS context methods and variables
+---
+
+DBOS provides a number of context methods and variables you can use to interact with functions and the database.
+All are accessed through the syntax `DBOS.<method>` and can only be used once a DBOS class object has been initialized.
+
+## Context Methods
+
+### send
+
+```python
+DBOS.send(
+    destination_id: str,
+    message: Any,
+    topic: Optional[str] = None
+) -> None
```

Comment:
Is this legal MD?

Reply:
Yeah, it's legal, not sure why GH doesn't like it but it renders correctly